# Define the goal
geometry_msgs/PoseStamped target_pose
---
# Define the result
---
# Define a feedback message
string description
float32 distance
